get_involved.md

```diff
@@ -20,25 +22,27 @@ There are plenty of opportunities for you to contribute to the HSF, be it by
 participation in technical discussions, by joining one of our working groups, or
 by joining the HSF with your software project:
 
-  *  Participation in HSF activities and forums. See the Activities page.
-  *  Participation in HSF events and meetings.
-  *  Contributing to the [HEP S&C Knowledge Base](http://hepsoftware.org)
-  *  Taking advantage for your project from the HSF (to be created)
-  *  Giving input to the HSF startup team
+  *  Participation in HSF [working groups](/what_are_WGs.html) and activities
```

Comment:
Did you not mean to say "and activity areas" and link to `what_are_activities.html`?